packages/legend-graph/src/models/metamodels/pure/valueSpecification/PropagatedMilestoningParameterValue.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ValueSpecification,
+  type ValueSpecificationVisitor,
+} from './ValueSpecification';
+
+export class PropagatedMilestoningParameterValue extends ValueSpecification {
+  isDefaultDatePropagationSupported!: boolean;
```

Comment:
I kinda want this to be more like `PropagatedValue extends ValueSpecification`, but let's see.

Reply:
Also, we have a convention for this kind of stuff to prefix it with `INTERNAL__`